verify/README

```diff
@@ -2,77 +2,91 @@
 
 This is the deployment container and configuration for the Cockpit
 integration tests. This documentation is for deployment on Fedora 22+
-or RHEL 7+
+or RHEL 7+.
 
-This is for a single operating system. Choose the operating system to
-test ahead of time.
+Use the following commands to run the verify container as a one off:
 
-## How to deploy
+   $ sudo yum -y install docker atomic
+   $ sudo systemctl start docker
+   $ sudo atomic run cockpit/infra-verify
```

Comment:
You can use ```atomic``` stuff to avoid the massive long docker command. You can use ```atomic run``` without ```atomic install```.